event/event.go

```diff
@@ -102,6 +164,7 @@ func (r *RawSubscription) Serve(handler RawMessageHandler) error {
 			}()
 			err := handler(msg.Body)
 			if err != nil {
+				slog.Error("message handler failed", "message", string(msg.Body), "error", err)
```

Comment:
I dont like logging in libraries, but in some cases the error will be produced internally by the library, the user code won't even be called, so there will be no opportunities to log anything.